omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,111 @@
+Administrators with restricted permissions
+==========================================
+
+
+Summary
+-------
+
+An administrator with restricted permissions is a way to cater for the need for more powerful users. In the real world, these will typically be imaging facility managers, image analysts, or anybody who needs to organize users and data in OMERO. The only other way how to create users with necessary powers for abovementioned actions would be to create new administrators in OMERO, which is not advisable for security reasons. Administrators with restricted permissions can be created and and their permissions tailored for the task they are meant to perform on behalf of other OMERO users.
+
+Typically, a full administrator in OMERO would create new administrators with restricted permissions using OMERO.web interface (hook to Help with Webadmin graphic screenshots)
+
+We suggest here 4 setups that should cover the 4 most mainstream workflows. Please note that it is fully possible to combine the permissions (check the checkboxes in the OMERO.web interface) in any way you see fit. The permission groupings were designed in such a way that they still bear useful functionality even when used in isolation. For example checking the Chown checkbox will indeed give the new administrator with restricted permissions the power to change (transfer) ownership of other users' data with no other ifs and buts. Nevertheless, please od not expect for any workflows mentioned here that all OMERO.clients (OMERO.web, OMERO.insight, command line interface (CLI)) are fully equipped to execute them (see details below). New features will be added in OMERO.clients in 5.3.x series of OMERO releases.
+
+Note on group allegiance: All the workflows here do not assume that the administrator with restricted permissions is not a member of any group except the System group. This does not preclude such administrator from being a member of any number of groups though, just as any other user, and still be able to perform his/her workflows dealing with other users' data in the groups which he/she is not a member of. See note on OMERO.insight in the Worfkflow 1 below though (hook to Workflow 1).
```

Comment:
I think you've got a double negative: I think you want "the workflows here assume that..." OR "administrator with restricted permissions is a member of..."